.github/workflows/pull-noir.yml

```diff
@@ -1,4 +1,4 @@
-# Create a pull request from current Noir master.
+# Create a pull request from current Noir nightly.
```

Comment:
I dont know if you guys still want a workflow to make PRs into noir with the patch, but otherwise we can delete the other side of the sync

Reply:
I thought opening a PR could involve some mechanical steps, such as rebasing on the remote branch _without_ the fixup and patch marker commit, which could benefit from automation, even if it's just manually triggered, but we should definitely disable any automation for now. I thought we can return to this question once we actually have a situation and understand better what's a nice way to do it. @TomAFrench ?

Reply:
Yeah, I'm fine with doing this manually in the short term and automating it later.